### ISSUE TITLE
Keyboard navigation is not working properly

### ISSUE THREAD

**Issue by gyurmey2** [fancyapps/ui#583]
### Describe the bug

The tab stop is not working properly. After going to the thumbnails, it only comes to specific a point and then goes back to the toolbar.

### Reproduction

https://jsfiddle.net/n84fxag7/1/

**Comment by fancyapps** [comment]
Hi,

Thanks for the report, it should be improved in the latest update.

**Comment by gyurmey2** [comment]
Hi @fancyapps, thanks for the update. However, it seems that navigating through the thumbnails with the keyboard [ends at the viewport](https://jsfiddle.net/9e3y8xtc/).

**Comment by fancyapps** [comment]
I don't know what you expected, but this is what I think should be right, and I also think it meets the accessibility guidelines

**Comment by gyurmey2** [comment]
I thought I should keep scrolling, but maybe you're right. Thanks!
